runtime/doc/eval.txt

```diff
@@ -10516,5 +10516,124 @@ This is not allowed when the textlock is active:
 	- closing a window or quitting Vim
 	- etc.
 
+==============================================================================
+13. Command-line expressions coloring			*expr-coloring*
+
+Expressions entered by user in |i_CTRL-R_=|, |c_CTRL-\_e|, |quote=| are 
+colored by built-in expressions parser.  It uses highlight groups described in 
+the below table, which may be overriden by user colorschemes, all linked to 
```

Comment:
"below table" => "table below" (can be both ways, but latter is a bit more Englishy)